### PR TITLE
Moves the spare ID inside the captain's locker

### PR DESCRIPTION
**Pull request by Spess-Rando**
:cl: Slignerd
balance: Following an immense number of complaints filed by security and command personnel, the Captain's spare ID will from now on be placed inside his locker. We fail to see how this would help the Captain access the spare in the event he lost his ID, but the complainants have been VERY insistent.
/:cl:

The possibility to acquire the spare ID illicitly still exists, but with this change:

- The high reward item (if not *the highest* reward item) that is the captain's ID requires more effort to get.
- Forcing the thief to not only hack two/three airlocks open, but also steal the locker and break it open is just enough to give others a decent chance to intervene.
- Rounds that don't start with a Captain or HoP aren't doomed to descend into all access madness within the first 5-15 minutes.

But also:

- Lowpop rounds without Captain or HoP aren't left screwed with no access at all.

### CODE REVIEW

**Comment by qbmax32** [comment]
I honestly believe this is a good change but the mass amounts of greytiders will be here soon to whine and stamp their feet

**Comment by Hatterhat** [comment]
just break it open with thrown spears or a fireaxe tbh

**Comment by PKPenguin321** [comment]
I used to hate this change because of how it hurt lowpop, but now that lockers can be broken open by anybody with enough determination I'm okay with it.

**Comment by evsey9** [comment]
just get a chainsaw its not that hard

**Comment by Poojawa** [comment]
Just make the Captain's door harder to break into, and give staff heads a button that's locked to, i dunno, bridge access so they can just walk in and grab it

**Comment by arsserpentarium** [comment]
How about situation,when there is no cap?

**Comment by Bawhoppen** [comment]
The locker can be smashed open.

**Comment by king5327** [comment]
The spare exists to provide access in the event that there's neither a Cap nor an HoP. If a head does not secure the spare then it deserves to be greytided. If there are no heads, doubly so.

There are three doors that need to be greytided to get in. Just set the cap's maint door and both inner bridge doors to metal reinforced. This prevents greytide from learning the power wire on less important doors and rushing through a minute later, since they have to greytide eng (or get lucky with budget gloves) first. On the other hand, heads can walk in through the bridge doors and hack the (unreinforced) cap's front door. Or thermite. Or use the AI.

Point being, the cap's spare has to be in a place that is possible to non-destructively enter. The only thing this PR will cause is captains losing their door remotes, sabers and gun to greytiders, not to mention the entire locker. All of which are easier to ignore when the greytider just ducks in and out before the doors close behind them.

**Comment by Bawhoppen** [comment]
But if someone has the spare, the gun and other stuff will be able to be taken anyways...

**Comment by ShiggyDiggyDo** [comment]
There's a top secret metastation technique that allows you to steal the spare in under a minute with no hacked doors

**Comment by DaxDupont** [comment]
Put it in wall safe

**Comment by Slignerd** [comment]
> If there are no heads, [the spare deserved to be greytided]

I disagree. Just because the Captain joins the round 5-15 minutes late should not mean complete all access anarchy. Neither should security or other departments be punished for the lack of command staff at roundstart.

The spare ID is a round-breaking item that gives you access to every single piece of equipment on the station. It should be awarded the first person who happens to have figured out wires and hack two airlocks.

> There are three doors that need to be greytided to get in. Just set the cap's maint door and both inner bridge doors to metal reinforced.

That just mean having to apply welder and crowbar. I might go with this suggestion, but _not_ as an alternative to the locker.

**Comment by ShiggyDiggyDo** [comment]
Give assistants who touch the spare constant brain damage

**Comment by zamolxius** [comment]
reinforce the captains office airlocks with plasteel.  no more hacky hacky. add the captain pet to captains room,  the angry naked collared assistant.  he bites if you come inside.  PvE feature 🔯🔯

**Comment by kevinz000** [comment]
i still think if someone can't secure the spare and all access gets handed out they need to get good.

**Comment by McBawbaggings** [comment]
all access is overrated
station mobility is at an all time high because you can smash nearly everything now and hacking isn't any more difficulty

the hand teleporter is far more useful anyway

**Comment by Slignerd** [comment]
Hard to "git gud" as the only officer on the station. You can't exactly stand and watch the captain's office non-stop. And when you don't watch, it takes an assistant a total of 20-30 seconds to hack in.

I'm thinking it's more like, if someone wants access to everything on the station, it should require some more effort them.

**Comment by AlexTheSysop42** [comment]
if you're the only officer and there's no command staff, just ask the AI to let you in and secure the spare yourself

**Comment by Spaceinaba** [comment]
anyone who disagrees with this PR is unrobust

**Comment by Slignerd** [comment]
@AlexTheSysop42 You do realize that "the only officer" is a lowpop situation, and that a lowpop situation often does not have an AI?

**Comment by Plapatin** [comment]
i dont know how to feel about this, however stuffing the captain's spare in his locker seems rather unnecessary, especially considering you dont need much more then a decent brute weapon and a bit of determination to access ALL of the cap's stuff with this change instead of just his spare
if you ask me, it might be a good idea to give reinforced doors more attention by adding plasteel or similar as a cover to the wires to the cap's doors.
barring that idea giving the spare it's own display case but it can actually be unlocked by the captain _AND JUST THE CAPTAIN ALONE_ so it gives sec a moment to stare at the BEEPBEEPBEEP and fire alarm blaring in the captain's office and go "oh shit spare stealer" - this also makes heads communicate with sec, and communication is always a good thing

**Comment by AlexTheSysop42** [comment]
I guess I did forget about that, but the officer could potentially hack in himself as well

**Comment by Cyberboss** [comment]
If you don't want your spare taken, fucking move it. Why is it so hard?

**Comment by Slignerd** [comment]
@Cyberboss It seems that everything I've said in this conversation just went over your head completely.

**Comment by Slignerd** [comment]
Let me reiterate this for clarity.


- There are many rounds that don't have a Captain, HoP, or an AI, or a signigicant security force.
- These round should be completely fucked with all access greytide for everyone to the absence of these roles
- Reinforced airlock suggestion, while decent, should in no way *replace* the locker change, since pretty much all it does is force the thief to apply a welder and a crowbar, which pretty much every tool-abusing assistant has.

Are these really so difficult to process? Do we have to constantly go through the circular and pointless discussion about what the AI or the Captain should do?

Then I have to reiterate my point about their absence, and then you just shove the responsibility of other roles, and then I describe that the other roles are not equipped to handle that responsibility.

It goes on and on.

So, TL;DR, stop trying to start circular arguments because you're upset that a PR makes all access greytiding take some goddamn effort.

**Comment by ExcessiveUseOfCobblestone** [comment]
Is there anything wrong with all access on lowpop?

Why would you even play lowpop sec if you're going to whine about people doing crimes on lowpop which arguably aren't even a crime if you consider that they have to mold the situation to fit being in a ~30 person-intended station with 10 people.

**Comment by Slignerd** [comment]
If your fellow crewmembers also see it that way, you're welcome to start busting that locker together.

**Comment by Potato-Masher** [comment]
I like this change, and it should deter some non-antag thefts. Then again, some people are still going to try and steal it anyways, since this is all access we are talking about.

Best course of action next would probably just to increase door security on the caps office. Logically the Captain of the station would want to have his office and his bedroom locked up from unwanted visitors.

**Comment by YogsMcree** [comment]
This change + the increased hacking difficulty is going to extend rounds on bagil wonderfully. Its annoying that vanilla traitors rounds end in 20 minutes because greytide have AA and hoarded sec / rnd gear.

**Comment by BeeSting12** [comment]
Bad change. 👎

**Comment by Spaceinaba** [comment]
this thread is a honeypot for greytiders

**Comment by Slignerd** [comment]
@BeeSting12 Bad comment. 👎

**Comment by BeeSting12** [comment]
@Slignerd bad comment on my comment :-1:

**Comment by ShizCalev** [comment]
Not a fan of this, however I do like that you added the captain's hat and carapace to his locker, that's actually a nice change (even though you didn't really state it anywhere.)

**Comment by epochayur** [comment]
Good PR. I'd honestly like to see the spare removed entirely, but this works in the interim.

_No, hacking 3 doors should not be enough to gain all access._ 

It would take someone MAYBE 60 seconds to get a screwdriver/multitool/gloves and find the power wire, then an additional 15 seconds to hack the last few doors between the hall and the captain's office. Seems like a pitiful amount of work to get one of the most powerful/convenient items in the game. Anyone crying about this is a greyshirt who's sad they don't get an instant "get into armory free" card. The "but lowpop access" argument is not an argument either. That's what the HoP/AI/silicons are for. Yeah, the locker takes a little more effort to get into, but it's more effort than none, and any deterrent to keep random assholes coming in and getting access to every single part of the station in the first 2 minutes of the game is a good change.

**Comment by Qbopper** [comment]
i had a sperg out in discord about this but i wanted this change a while ago and anyone who disagrees because "lol it doest change anything they can just take the locker" are missing the point and i'mm going to go lightm yself on fire now

**Comment by ExcessiveUseOfCobblestone** [comment]
I thought this was to do with lowpop? If it's normalpop/high then yeah it should only take 3 doors when heads should be securing it.

**Comment by CPTANT** [comment]
Let me guess, the secret technique is reversing the disposals pipe....

**Comment by ExcessiveUseOfCobblestone** [comment]
it's not very secret if you say it out loud (or if you have to crowbar, weld, wrench, rotate, wrench, weld, and replace tile in the middle of the hallway before anyone notices).

**Comment by duncathan** [comment]
good change imo this is nice

**Comment by CPTANT** [comment]
This is just stupid.
